assets-styleguide/js/styleguide.js

```diff
@@ -114,6 +121,17 @@ $('.sidenav').on('click', 'a', function(e) {
       } else if (window.location.hash !== newHash) {
         window.location.hash = newHash;
       }
+    },
+    done: function () {
```

Comment:
I think this would require more research. We have an accessibility expert on staff, I'll try and ask him today